tests/test_discovery.py

```diff
@@ -514,6 +515,22 @@ def test_patch(self):
 
     self.assertEqual(request.method, 'PATCH')
 
+  def test_batch_request_from_discovery(self):
+    self.http = HttpMock(datafile('zoo.json'), {'status': '200'})
+    # zoo defines a batchPath
+    zoo = build('zoo', 'v1', http=self.http)
+    expected = "https://www.googleapis.com/batchZoo"
```

Comment:
Drop "expected" as a local field and just use this literal value directly at the field's one use-site.